mino/minows/integration_test.go

```diff
@@ -45,7 +45,7 @@ package minows
 // }
 //
 // func newMinowsAndRPC() (mino.Mino, mino.RPC) {
-// 	m, err := NewMinows("ip4/127.0.0.1/tcp/9998/ws", "ip4/127.0.0.1/tcp/9998/ws",
+// 	m, err := newMinows("ip4/127.0.0.1/tcp/9998/ws", "ip4/127.0.0.1/tcp/9998/ws",
```

Comment:
For the whole file: please don't commit commented out code. Remove the commented code from the commit.

Reply:
removed file